src/contexts/CalenarProvider.tsx

```diff
@@ -1,19 +1,44 @@
 import React, { createContext, useState } from 'react';
-import { SchedulesData } from '@/components/SchedulesPage/Schedules';
 
+//import { SchedulesData } from '@/components/SchedulesPage/Schedules';
+
+export interface SchedulesData {
+  id: number;
+  title: string;
+  content: string;
+  startDateTime: string;
+  endDateTime: string;
+  author?: {
+    name: string;
+    imageUrl: string;
+    role: string;
+    grade: string;
+    username: string;
+    createdDate: string;
+  };
+  teamResponse?: {
+    id: number;
+    name: string;
+    description: string;
+    color: string;
+  };
```

Comment:
제가 Author하고 Team 타입 정의해둔 거 있어서 나중에 import해오셔서 쓰시면 될 것 같아용
```suggestion
  author?: Author
  teamResponse?: Team
```